apps/dashboard/src/components/workflow-editor/steps/email/novu-branding.tsx

```diff
@@ -28,7 +27,8 @@ export const NovuBranding = ({ className, ...rest }: NovuBrandingProps) => {
   const removeNovuBranding = organizationSettings?.data?.removeNovuBranding;
   const isUpdating = updateOrganizationSettings.isPending;
 
-  if (removeNovuBranding) return null;
+  // Don't render anything while loading or if branding should be removed
+  if (isLoadingSettings || removeNovuBranding) return null;
```

Comment:
Prevents flicking the branding for a split second while its loading and branding is disabled.